### ISSUE TITLE
F5(debug) do and prints nothing

### ISSUE THREAD

**Issue by zelyony** [WebFreak001/code-debug#191]
Windows x64 10
VSCode latest with today installed code-d and "D Language utility ext pack" 
mago-mi.exe copied form Baggins's dlangide to DMD\bin that is in PATH

I set breakpoints in code - red circles in right side editor.
do F5 or press button |> - and nothing happens.
dub build app built "debug" version (see in terminal). I can run app in terminal too.

dub clean + delete manually EXE: F5 do nothing, tell nothing (something like "I cannot find EXE")

how to debug app and breaks in breakpoints?

probably add to code-debug some OutputDebugString(..) for Windows as some status: what doing, which commands..
such

**Comment by zelyony** [comment]
https://forum.dlang.org/post/iplwvbwevtrqlmexniiv@forum.dlang.org
when we'll solve problem I'll update forum post

**Comment by zelyony** [comment]
"output" for "code-d & serve-d" in vscode:

Server: 2019-09-19T14:22:34.189:dsymbol\src\dsymbol\modulecache.d:existanceCheck:60 Cannot cache modules in C:\D\dmd2\src\phobos because it does not exist

Server: 2019-09-19T14:22:34.189:src\dcd\server\main.d:main:289 Request processed in 0 milliseconds

2019-09-19T14:22:34.194:source\served\extension.d:changedConfig:192 Finished config change of file:///c%3A/temp/D/vscode1 with 3 changes in 3 secs, 697 ms, 383 μs, and 8 hnsecs.


why Server searching something in C:\D\.. I used another path. and Server can find write directory through 
```
C:\programz\D>where dmd
C:\programz\D\dmd2\windows\bin\dmd.exe
```

**Comment by WebFreak001** [comment]
code-d doesn't have anything to do with this, please make an issue there if you think something is broken in the D feature-set. (not debugging) It's just searching in `C:\D` because that's the default path and the configured path (in `"d.stdlibPath"`, but if you want to discuss more about that, make an issue in code-d)

Did you configure your project properly and compile in debug mode? Please check the README if you have done everything properly and send your project configuration afterwards if it still doesn't work.

**Comment by WebFreak001** [comment]
duplicate of #189

Use the C/C++ Extension which ships with visual studio debugging capabilities instead for now.

**Comment by zelyony** [comment]
well, README doesn't help to much (nothing interesting)

i added/changed in config next lines:
`    "magomipath": "C:/programz/D/dmd2/windows/bin/mago-mi.exe",
`NB "magomipath" should be FULL. just "mago-mi", "mago-mi.exe" is not working, nothing running. also mago-mi is visible in my PATH.
`    "target": "./vscapp1.exe",`
instead "target": ".bin/executable" write real EXE-name else nothing happens

after such steps app is running, console with "hello world" showing but breakpoints is not working
DEBUG CONSOLE says:
```
Starting program: C:\temp\D\vscapp1\vscapp1.exe
Loaded.
Running executable
Invalid breakpoint parameters // ????????????
Execution is not paused
```

I also installed "C/C++ ms-vscode.cpptools", create new config for it and yes, local watches, breakpoints, call-stacks are working.
BUT again M$ shit problem:
when I use VSCodium (same VSCode w/o telemetry) I've got (its not code-d problem, just info)
```
You may only use the C/C++ Extension for Visual Studio Code with
Visual Studio Code, Visual Studio or Visual Studio for Mac software
to help you develop and test your applications.

```
